rest/route/permissions.go

```diff
@@ -0,0 +1,42 @@
+package route
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/evergreen-ci/evergreen"
+	"github.com/evergreen-ci/evergreen/rest/model"
+	"github.com/evergreen-ci/gimlet"
+)
+
+type permissionsGetHandler struct{}
+
+func (p *permissionsGetHandler) Factory() gimlet.RouteHandler                     { return p }
+func (p *permissionsGetHandler) Parse(ctx context.Context, r *http.Request) error { return nil }
+
+func GetPermissions(permissions []string) []model.APIPermission {
+	out := []model.APIPermission{}
+
+	for _, key := range permissions {
```

Comment:
should this logic be in a `BuildFromService`?

Reply:
still learning the paradigms of the API, but it looks like BuildFromService is used to convert service levels structs to an API struct. not sure that's what happening here tho? @tychoish recommended making these public functions into private methods on the `permissionGetHandler` type which sounds like the right direction.

Reply:
There isn't a db model, so I think the best place for this logic is here